GDJS/Runtime/runtimegame.ts

```diff
@@ -715,24 +729,37 @@ namespace gdjs {
           return;
         }
 
+        const now = Date.now();
+        notYetSentDuration += now - lastSessionResumeTime;
+        lastSessionResumeTime = now;
+
         // Group repeated calls to sendSessionHit - which could
         // happen because of multiple event listeners being fired.
-        if (Date.now() - lastSessionHitTime < 3 * 1000) {
+        if (notYetSentDuration < 5 * 1000) {
           return;
         }
-        lastSessionHitTime = Date.now();
+        const onlySeconds = Math.floor(notYetSentDuration / 1000) * 1000;
```

Comment:
I'm really struggling to understand what the variable represents as the naming is really generic.
May I suggest `notYetSentDurationWithOnlySeconds`?